### PR TITLE
Replaced 'repr' with 'srepr' in the tutorial 

### LINKED ISSUE #10185: Clarification about srepr in the docs
Quoting from Gitter chat

> srepr gives the “repr” form, which is what reprwould normally give
but for SymPy we don’t actually use it for `__repr__` because it’s way too verbose
and on top of that when you print a list or a tuple it calls repr on the elements instead of str, so a list of sympy expressions would always have the repr output
that page should probably mention all of this. it could be a lot clearer
oh I guess that’s the tutorial. For the tutorial I would just replace the “repr” heading with “srepr”. But in the docs for srepr it should have the discussion about the repr builtin

We should rename the `repr` section in the tutorial to `srepr`, and add some in-depth discussion (basically what I wrote above) to the `srepr` docstring, which appears [here](http://docs.sympy.org/dev/modules/printing.html?highlight=srepr#sympy.printing.repr.srepr).

### PR DESCRIPTION
**Pull request by sampadsaha5**
Fixes #10185

### CODE REVIEW

**Comment by sampadsaha5** [comment]
@moorepants 

Is this ok now?

**Comment by sampadsaha5** [comment]
ping @asmeurer @leosartaj

**Comment by scopatz** [comment]
This looks good to me.

File: doc/src/tutorial/printing.rst [hunk 1]
```diff
@@ -231,8 +231,8 @@ printer.
 
 .. rubric:: Footnotes
 
-.. [#srepr-fn] SymPy does not use the Python builtin ``repr()`` function for
-   repr printing, because in Python ``str(list)`` calls ``repr()`` on the
+.. [#srepr-fn] SymPy does not use the Python builtin ``srepr()`` function for
```
Comment (gxyd) [inline doc/src/tutorial/printing.rst]: Is there is no `srepr()` built into Python?

**Comment by gxyd** [comment]
As was mentioned to me on gitter chat.
> For the tutorial I would just replace the “repr” heading with “srepr

So i guess, it seems its just about replacing `repr` with `srepr` just in heading?

File: doc/src/tutorial/printing.rst (hunk 1 above)
Comment (asmeurer) [inline doc/src/tutorial/printing.rst]: This footnote should have stayed unchanged.

**Comment by asmeurer** [comment]
This doesn't completely fix the issue because there is no discussion on the srepr docstring (this is fine, I'm just noting this so that we don't close the issue when this is merged).

File: doc/src/tutorial/printing.rst (hunk 1 above)
Comment (sampadsaha5) [inline doc/src/tutorial/printing.rst]: @asmeurer 
Does this footnote should have stayed unchanged? That means `srepr` back to `repr`.

**Comment by sampadsaha5** [comment]
@asmeurer 

I think now this would be fine.

**Pull request merged**